doc/source/getting_started/launcher.rst

```diff
@@ -0,0 +1,228 @@
+
+
+.. _ref_launch_pymapdl:
+
+
+Launch PyMAPDL
+==============
+
+PyMAPDL can start MAPDL locally, or connect to a session already running locally or in a remote machine.
+
+* `Launch PyMAPDL with a local MAPDL instance`_
+* `Connect PyMAPDL to a local MAPDL instance`_
+* `Connect PyMAPDL to a remote MAPDL instance`_
+
+If you have any problem launching PyMAPDL, see :ref:`Launching issues <ref_launching_issue>`.
+
+
+Launch PyMAPDL with a local MAPDL instance
+------------------------------------------
+
+You can use the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`
+method to have Python start MAPDL and automatically connect to it:
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl()
+    >>> print(mapdl)
+    Product:             Ansys Mechanical Enterprise
+    MAPDL Version:       24.1
+    ansys.mapdl Version: 0.68.0
+
+
+This is the easiest and fastest way to get PyMAPDL up and running. 
+But you need to be able to launch MAPDL locally.
+
+If PyMAPDL cannot find your local installation, please refer to
+`Setting MAPDL location in PyMAPDL`_.
```

Comment:
```suggestion
If PyMAPDL cannot find your local installation, see
`Setting MAPDL location in PyMAPDL`_.
```
Omit "please" in technical documentation. It helps readers if you always introduce the link with "see" rather than go to or visit.

Reply:
Thank a lot... I should remember next time!